PLAN.md

```diff
@@ -0,0 +1,52 @@
+- Create own directory plugin?
+    - netrw not lightweight, pollutes clipboard history
+
+- Override window navigation commands
+    - File explorer window
+        - Fixed window to the left
+        - Window navigation
+            - http://blog.paulrugelhiatt.com/vim/2014/10/31/vim-tip-automatically-create-window-splits-with-movement.html
+            - Command: `oni.window.moveLeft`, `oni.window.moveRight`, `oni.window.moveUp`, `oni.window.moveDown`
+            - WindowManager - multiple Neovim editor instances
+
+- Window focus management
+    - Create Window Manager
+    - Have focus managed
+    - Pull keyboard out externally, have focused window handle input
+        - Keyboard focus for active window
+    - Create `deactivate`, `activate` concepts for editor
+    - Push `onKeyDown` to editor
+    - How to keep from 'drilling in' to file?
```

Comment:
I agree with you, if I open a folder inside of the folder pane, I don't want it to open in the folder pane, that would be a pain.

I would want it to open in the "active editor", or split if I use c-v/c-s starting from the most far right/far bottom (some users like left and top, in fact vim's standard is left and and top which I never understood but... we could check this value)